LayoutTests/fast/canvas/canvas-imagedata-dimension-no-overflow-expected.txt

```diff
@@ -0,0 +1,24 @@
+CONSOLE MESSAGE: [object ImageData]
```

Comment:
You don't need to connsole log anything like these.

Reply:
Fixed